src/vs/workbench/api/common/extHostTerminalService.ts

```diff
@@ -899,19 +905,31 @@ class EnvironmentVariableCollection {
 		return scopedCollection;
 	}
 
-	replace(variable: string, value: string, scope: vscode.EnvironmentVariableScope | undefined): void {
-		this._setIfDiffers(variable, { value, type: EnvironmentVariableMutatorType.Replace, scope });
+	replace(variable: string, value: string, options: vscode.EnvironmentVariableMutatorOptions | undefined, scope: vscode.EnvironmentVariableScope | undefined): void {
+		if (this._extension && options) {
+			isProposedApiEnabled(this._extension, 'envCollectionOptions');
```

Comment:
@Tyriar This seems to return a boolean but isn't consumed anywhere, did we mean to use `checkProposedApiEnabled` instead?

Reply:
I've changed it in https://github.com/microsoft/vscode/pull/183255.